server/test/controllers/ti/TrustedIntermediaryControllerTest.java

```diff
@@ -0,0 +1,106 @@
+import java.util.Optional;
+import models.Applicant;
+import models.TrustedIntermediaryGroup;
+import org.junit.Before;
+import org.junit.Test;
+import play.mvc.Http;
+import play.mvc.Result;
+import play.test.Helpers;
+import repository.UserRepository;
+
+public class TrustedIntermediaryControllerTest extends WithMockedProfiles {
+  UserRepository repo;
+  TrustedIntermediaryController tiController;
+  ProfileFactory profileFactory;
+  CiviFormProfileData data;
+
+  @Before
+  public void setup() {
+    profileFactory = instanceOf(ProfileFactory.class);
+    tiController = instanceOf(TrustedIntermediaryController.class);
+    Applicant managedApplicant = createApplicant();
+    createTIWithMockedProfile(managedApplicant);
+    repo = instanceOf(UserRepository.class);
+    data = profileFactory.createFakeTrustedIntermediary();
+  }
+
+  @Test(expected = NullPointerException.class)
+  public void addApplicantTestWithMissingDob() {
+    Http.RequestBuilder requestBuilder =
+        addCSRFToken(
+            Helpers.fakeRequest()
+                .bodyForm(
+                    ImmutableMap.of(
+                        "firstName",
+                        "first",
+                        "middleName",
+                        "middle",
+                        "lastName",
+                        "last",
+                        "emailAddress",
+                        "sample@fake.com")));
+    TrustedIntermediaryGroup group = repo.listTrustedIntermediaryGroups().get(0);
+    Result result = tiController.addApplicant(group.id, requestBuilder.build());
+    assertThat(result.flash().get("error")).isEqualTo("providedDateOfBirth cannot be null");
+  }
+
+  @Test
+  public void addApplicantTestWithAllInformation() {
+    Http.RequestBuilder requestBuilder =
+        addCSRFToken(
+            Helpers.fakeRequest()
+                .bodyForm(
+                    ImmutableMap.of(
+                        "firstName",
+                        "first",
+                        "middleName",
+                        "middle",
+                        "lastName",
+                        "last",
+                        "emailAddress",
+                        "sample2@fake.com",
+                        "dob",
+                        "2022-07-18")));
+    TrustedIntermediaryGroup group = repo.listTrustedIntermediaryGroups().get(0);
+    Result result = tiController.addApplicant(group.id + 1, requestBuilder.build());
+    assertThat(result.status()).isEqualTo(SEE_OTHER);
+    Optional<Applicant> testApplicant =
+        repo.lookupApplicantByEmail("sample2@fake.com").toCompletableFuture().join();
+    assertThat(testApplicant.get().getApplicantData().getDateOfBirth().get().toString())
+        .isEqualTo("2022-07-18");
+  }
+
+  @Test
+  public void testUpdateDOBFunctionWithExistingDob() {
+    Http.RequestBuilder requestBuilder =
+        addCSRFToken(Helpers.fakeRequest().bodyForm(ImmutableMap.of("dob", "2022-10-05")));
```

Comment:
Can you move this experssion to line 98 so that when the test reads is 

1. Initialize form and create new applicant.
2. Initialize change-dob request and update DOB.